assets/prototype/domain/shared/predicates/tickets/filters/expiredOnly.ts

```diff
@@ -0,0 +1,10 @@
+/**
+ * Internal dependencies
+ */
+import isExpired from '../isExpired';
+
+const expiredOnly = (ticketEntities: any[]) => {
+	return ticketEntities.filter((ticketEntity) => isExpired(ticketEntity));
```

Comment:
```suggestion
	return tickets.filter((ticket) => isExpired(ticket));
```

would need to adjust the function declaration too